wdl/tasks/PBUtils.wdl

```diff
@@ -258,6 +258,48 @@ task ExtractHifiReads {
     }
 }
 
+task RemoveKinetics {
+    input {
+        File bam
+        String prefix = "no_kinetics"
```

Comment:
I'd advocate for computing the new BAM name without an explicit prefix.
I'd vote for something like:

`[a-z]+.bam` to `[a-z]+.no_kitentics.bam`